performance/tasks/cumul_smoke.yaml

```diff
@@ -0,0 +1,143 @@
+{% set flavor = flavor or "m1.tiny" %}
+{% set image = image or "^cirros.*uec$" %}
+{% set availability_zone = availability_zone or "nova" %}
+{% set start_cidr = start_cidr or "10.2.0.0/24" %}
+{% set ext_network = ext_network or "public" %}
+---
+
+  KeystoneBasic.create_and_delete_role:
+    -
+      runner:
+        type: "constant"
+        times: 1
+        concurrency: 1
+
+  NovaServers.boot_and_delete_server:
+    -
+      args:
+        flavor:
+            name: "{{flavor}}"
+        image:
+            name: "{{image}}"
+        auto_assign_nic: true
+      runner:
+        type: "constant"
+        times: 1
+        concurrency: 1
+      context:
+        users:
+          tenants: 1
+          users_per_tenant: 1
+        network:
+          start_cidr: "{{start_cidr}}"
+
+  NovaSecGroup.create_and_delete_secgroups:
+    -
+      args:
+        security_group_count: 2
+        rules_per_security_group: 2
+      runner:
+        type: "constant"
+        times: 1
+        concurrency: 1
+      context:
+        users:
+          tenants: 1
+          users_per_tenant: 1
+
+  NovaKeypair.create_and_delete_keypair:
+    -
+      runner:
+        type: "constant"
+        times: 1
+        concurrency: 1
+      context:
+        users:
+          tenants: 1
+          users_per_tenant: 1
+
+  CinderVolumes.create_and_delete_volume:
```

Comment:
remove this scenario, all atomics are collect in next one:
https://github.com/openstack/rally/blob/master/rally/plugins/openstack/scenarios/cinder/volumes.py#L388-L404